Makefile

```diff
@@ -179,15 +178,18 @@ TARGET_ARCH_DEBIAN = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET
 TARGET_ARCH_REDHAT = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t redhat)
 TARGET_ARCH_APPIMAGE = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t appimage)
 TARGET_ARCH_ELECTRON_BUILDER = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t electron-builder)
+TARGET_PLATFORM_PKG = $(shell ./scripts/build/target-convert.sh -r $(TARGET_PLATFORM) -t pkg)
```

Comment:
What :| That is really weird. I triple checked, and the script is there on my local tree, for the latest master:

```
$ git log -1
commit e52ef501bbcefe7d90bc760ef829211aefeb9fc3 (HEAD -> master, origin/master)
Author: Juan Cruz Viotti <jv@jviotti.com>
Date:   Tue Jul 11 17:12:44 2017 -0400

    chore: avoid brew taking a long time on macOS Travis builds (#1583)

    - Homebrew now auto-updates before performing any installation, which
      can take quite some time. We can disable this by enabling
      HOMEBREW_NO_AUTO_UPDATE

    - We can pass the --force-bottle option to `brew install` to force it to
      download pre-compiled binaries, and don't attempt any compilations

    See: https://github.com/resin-io/etcher/pull/1582
    Change-Type: patch
    Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>

$ ls -l scripts/build
total 96
-rwxr-xr-x  1 jviotti  staff  2278 Jul  6 15:39 architecture-convert.sh*
-rwxr-xr-x  1 jviotti  staff  1276 Jun  6 12:59 check-dependency.sh*
-rwxr-xr-x  1 jviotti  staff  1476 Jun 12 15:22 dependencies-npm-extract-addons.sh*
-rwxr-xr-x  1 jviotti  staff  4021 Jul 10 11:21 dependencies-npm.sh*
drwxr-xr-x  8 jviotti  staff   272 Jul 11 15:07 docker/
-rwxr-xr-x  1 jviotti  staff  2061 Mar 30 18:05 download-tool.sh*
-rw-r--r--  1 jviotti  staff  2670 Mar 31 13:17 electron-brand-exe-win32.sh
-rwxr-xr-x  1 jviotti  staff  2670 Mar 31 13:17 electron-brand-exe.sh*
-rwxr-xr-x  1 jviotti  staff  1789 Mar  9 16:32 electron-sign-exe-win32.sh*
-rwxr-xr-x  1 jviotti  staff  1247 Mar 31 13:17 electron-sign-file-darwin.sh*
-rwxr-xr-x  1 jviotti  staff  1125 Apr 24 10:57 tar-gz-file.sh*
-rwxr-xr-x  1 jviotti  staff  1390 Jun 30 12:34 target-convert.sh*
-rwxr-xr-x  1 jviotti  staff  1629 Jun 27 12:52 zip-file.sh*
```

I'll investigate...